sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/SpanExporterConfiguration.java

```diff
@@ -59,6 +63,21 @@ static OtlpGrpcSpanExporter configureOtlpSpans(ConfigProperties config) {
       builder.setTimeout(Duration.ofMillis(timeoutMillis));
     }
 
+    String certificate = config.getString("otel.exporter.otlp.certificate");
```

Comment:
the spec also suggests span and metric specific configuration. do we need to support that, or is this good enough (I think this is good enough for now...we can add the signal-specific ones later if needed)

Reply:
Yup missing the signal-specific ones for all these variables, will add in followup